### DISCUSSION TITLE
how to add a list of sites to the blocklist while the timer is running

### DISCUSSION THREAD

**Discussion by imflash217** [SelfControlApp/selfcontrol#746] [Q&A] [answered]
Hi,
I have a very long list of sites that I want to add to the blocklist while the timer is already running, but **its tedious to add each site individually**. 
Is there a way to add a `.txt` file to the blocklist?
Is there any format required?
Thanks

**Comment by cstigler** [comment, marked as answer]
Hi @imflash217 , you should be able to copy/paste in a list as long as it's separated by newlines. For example, make a `.txt` file that contains the following:
```
facebook.com
youtube.com
reddit.com
nytimes.com
cnn.com
```

then select all, and copy/paste it into SelfControl and it should add all of the sites there.

**Reply by imflash217** [reply]
Awesome. 🤩  Thanks 😊
